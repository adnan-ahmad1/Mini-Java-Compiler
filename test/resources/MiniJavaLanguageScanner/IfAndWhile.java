if (a < b) {

}

while (1 < 2) {

}