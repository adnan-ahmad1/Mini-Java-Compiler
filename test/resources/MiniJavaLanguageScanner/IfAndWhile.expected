IF LPAREN ID(a) LESS ID(b) RPAREN LCURLY RCURLY WHILE LPAREN INTEGER(1) LESS INTEGER(2) RPAREN LCURLY RCURLY